GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -42,36 +42,45 @@
   return storagePath;
 }
 
-@implementation GDTCORStorage
+@implementation GDTCORFlatFileStorage
 
 + (void)load {
   [[GDTCORRegistrar sharedInstance] registerStorage:[self sharedInstance] target:kGDTCORTargetCCT];
   [[GDTCORRegistrar sharedInstance] registerStorage:[self sharedInstance] target:kGDTCORTargetFLL];
   [[GDTCORRegistrar sharedInstance] registerStorage:[self sharedInstance] target:kGDTCORTargetCSH];
+
+  // Sets a global translation mapping to decode GDTCORStoredEvent objects encoded as instances of
+  // GDTCOREvent instead. Then we do the same thing with GDTCORStorage. This must be done in load
```

Comment:
Rename here.

Reply:
This comment refers to the old class name, so it's fine.